doc/guides/prog_guide/cnet.rst

```diff
@@ -0,0 +1,389 @@
+..  SPDX-License-Identifier: BSD-3-Clause
+    Copyright (c) 2022 Intel Corporation.
+
+.. _CNET_Library:
+
+CNET Stack Library
+==================
+
+Overview
+--------
+
+The CNET graph node library uses a set of graph nodes to create a UDP/TCP/IPv4 network stack in
+user space. One goal is to provide a network stack for applications in CNDP as the developer
+may have been using the Linux network stack via the socket interface.
+
+The CNET stack uses a socket like interface called channels to make it fimiliar to the developer,
+but does have a number of differences. One of the differences is that channels support multiple
```

Comment:
The sentence starting with "One of the differences..." is hard for me to understand. Suggest the following instead:

```One of the differences is that channels support sending multiple packets in a single transmission, and another is channels support receiving multiple packets at a time using a function pointer callback.```